From: Sébastien Noel <twolife@debian.org>
Description: fix FTBFS with GCC-16
Forwarded: no

--- a/src/shared/tools.h
+++ b/src/shared/tools.h
@@ -33,10 +33,7 @@ typedef unsigned long long int ullong;
 #define UNUSED
 #endif
 
-inline void *operator new(size_t, void *p) { return p; }
-inline void *operator new[](size_t, void *p) { return p; }
-inline void operator delete(void *, void *) {}
-inline void operator delete[](void *, void *) {}
+#include <new>
 
 #ifdef swap
 #undef swap
